docs/articles/documentation/getting-started/index.md

```diff
@@ -0,0 +1,195 @@
+It is recommended to create this directory as a subdirectory of the project you are going to test.
+
+This tutorial will demonstrate how to test not a developing project, but the published web page.
+To create tests for this page, create the *my-project* directory anywhere on your computer and the *tests* subdirectory in it.
+You can do this from a command line.
+
+```bash
+mkdir -p my-project\tests
+```
+
+## Installing TestCafe
+
+Make sure that Node.js and npm are installed on your computer. Then go to the tested project directory (*my-project*) and run a single command:
+
+```bash
+npm install --save-dev testcafe
+```
+
+This will create the *node_modules* directory in your current directory (if the first one doesn't exist yet) and will download the TestCafe package to that directory.
+
+## Writing Test
+
+### General Code Structure
+
+TestCafe organizes tests into fixtures. Test fixtures are JavaScript files that contain one or more tests for a single target URL.
+Therefore, you will usually have one test fixture per website page/route.
+
+To create a test fixture, just create a .js file in the *tests* directory and add code to the file.
+
+The following snippet illustrates basic elements of a TestCafe fixture:
+
+* fixture declaration - is added by using the the `fixture` keyword;
+* target webpage - is specified by using the `page` function;
+* test code - is placed within the `test` function.
+
+```js
+fixture `Fixture name`
+    .page('http://testcafe.devexpress.com/Example');
+
+test('Test 1 name', func t => {
+    //Test 1 code
+});
+
+test('Test 2 name', func t => {
+    //Test 2 code
+});
+```
+
+### Performing Actions on Page
+
+Every test should be capable of interacting with a page content. To perform user actions, TestCafe provides a number of methods: `click`, `hover`, `typeText`, `upload`, etc.
+They can be called in a chain.
+
+For example, the following fixture contains a simple test that types a developer name into a text editor and then clicks the Submit button.
+All test actions are implemented as functions of the test controller object `t`.  This object is used to access test run API.
+
+```js
+fixture `Getting Started - Test 1`
+    .page('http://testcafe.devexpress.com/Example');
+
+test('MyTest', async t => {
+    await t
+        .typeText('#Developer_Name', 'John Smith')
+        .click('#submit-button');
+});
+```
+
+For more information, see [Actions](../test-api/actions.md).
+
+### Assertions
+
+A functional test should also check the result of actions performed. For example, clicking the Submit button on the sample web page opens a "Thank you" page whose header addresses a user by the entered name.
+To check if the name is correct, you have to add an assertion to the test.
+
+TestCafe allows you to use any assertion library to perform verifications. Before calling assertions, you need to install the assertion library into your tested project.
```

Comment:
I think we shouldn't use Node's assert in the example. Firstly, the official Node documentation says that it "is not intended to be used as a general purpose assertion library".
Secondly, there is no problem to use an assertion from another library. 

It would be strange to use node's assert in the example and to say about some restrictions.